### PR TITLE
US 11 - Music App API music style generator now fetches new styles

### PR DESCRIPTION
**Pull request by IC-Socaciu**

### CODE REVIEW

File: pages/discover.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    []
+  );
+
+  const handleClick = () => {
+    mutate();
+  };
+
+  if (error) {
+    return <div>Error fetching data</div>;
+  }
```
Comment (FelineHuhn) [inline pages/discover.js:23]:
```suggestion

```

File: pages/discover.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+export default function Discover() {
+  const { data, error, mutate } = useSWR(
+    URL,
+    async (URL) => {
+      const response = await fetch(URL);
+      const data = await response.json();
+      return data;
+    },
+    []
```
Comment (FelineHuhn) [inline pages/discover.js:14]:
```suggestion
if (!response.ok) {
        throw new Error("Failed to fetch data");
      }

      const data = await response.json();
      return data;
    }
```

**Review by FelineHuhn — CHANGES_REQUESTED** [review]
Hey Ioana, nice progress and I love your new feature 🎉
I left you some suggested code snippets for a better error handling.

File: pages/discover.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import { styled } from "styled-components";
```
[the remark this answers is not in the document]
Comment (IC-Socaciu) [reply pages/discover.js:1]: Thank you. Corrected :)

**Pull request merged**